### ISSUE TITLE
ImportError: Building module CGAT._clusters2metrics failed

### ISSUE THREAD

**Issue by sebastian-luna-valero** [CGATOxford/cgat#172]
Hi,

There is an error when compiling cluster2metrics:
```
CGAT/_clusters2metrics.c:250:31: fatal error: numpy/arrayobject.h: No such file or directory
```

Details here:
https://travis-ci.org/CGATOxford/cgat/jobs/57975046

Could somebody please have a look?

Thanks!

**Comment by sebastian-luna-valero** [comment]
Should not this extension be compiled along with the other cython modules with `python setup.py develop`?

**Comment by AndreasHeger** [comment]
Indeed. And as a consequence the include paths are not set correctly. Cython extensions should be separate sub-packages (see Component). I will refactor.

**Comment by AndreasHeger** [comment]
fixed by e560f527aedda4069bdd481f4aec1224c7743ad1
